src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +51,19 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+        String email = "mate@gmail.com";
+        String password = "MateAcademy";
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+        User user = authenticationService.register(email, password);
+        System.out.println(authenticationService.login(email, password));
```

Comment:
Consider doing it like this:

![image](https://github.com/user-attachments/assets/b257c444-32db-43d3-8042-6b4f8353c3fe)